compiler/il/symbol/OMRResolvedMethodSymbol.cpp

```diff
@@ -1562,10 +1559,36 @@ OMR::ResolvedMethodSymbol::cannotAttemptOSR(TR_ByteCodeInfo bci,
    return false;
    }
 
+bool
```

Comment:
Is there any opportunity to share the impl of allCallerOSRBlocksArePresent with cleanupUnreachableOSRBlocks ?